vmdk_plugin/drivers/vmdk/vmdkops/esx_vmdkcmd.go

```diff
@@ -117,6 +117,11 @@ func (vmdkCmd EsxVmdkCmd) Run(cmd string, name string, opts map[string]string) (
 		break
 	}
 
+	if ans.buf == nil {
```

Comment:
_I do no think we can get here in failure case without printing ANY message from the loop above, and the logs did not have any message,_ so I do not see why this block is needed. Besides, with the changes suggested in this PR this can be impossible, will it ?  Besides, in cgo is it ok to compare go.nil with C (unsafe) ans.buf ? 

* if you want to for working on a repro, this makes sense, but for production code it feels not needed